packages/admin/blocks-admin/src/blocks/VimeoVideoBlock.tsx

```diff
@@ -0,0 +1,52 @@
+import { Field, FinalFormInput, FinalFormSwitch } from "@comet/admin";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { VimeoVideoBlockData } from "../blocks.generated";
+import { BlocksFinalForm } from "../form/BlocksFinalForm";
+import { createCompositeBlock } from "./factories/createCompositeBlock";
+import { createCompositeSettings } from "./helpers/composeBlocks/createCompositeSettings";
+import { BlockCategory } from "./types";
+
+export const VimeoVideoBlock = createCompositeBlock(
+    {
+        name: "VimeoVideo",
+        displayName: <FormattedMessage id="blocks.vimeoVideo" defaultMessage="Video (Vimeo)" />,
+        category: BlockCategory.Media,
+        blocks: {
+            settings: {
+                block: createCompositeSettings<Pick<VimeoVideoBlockData, "vimeoIdentifier" | "autoplay">>({
+                    defaultValues: { vimeoIdentifier: "", autoplay: false },
+                    AdminComponent: ({ state, updateState }) => {
+                        return (
+                            <BlocksFinalForm onSubmit={updateState} initialValues={state}>
+                                <Field
+                                    name="vimeoIdentifier"
```

Comment:
Should we add a validation here?

Reply:
You mean like for the youtube video block?

Reply:
Yes! To ensure that this is a valid Vimeo Identifier. Related #2086

Reply:
Done